typescript/sdk/src/utils/sealevel/tokenSerialization.ts

```diff
@@ -0,0 +1,170 @@
+import { PublicKey } from '@solana/web3.js';
+
+import { DomainId } from '@hyperlane-xyz/utils';
+
+/**
+ * Hyperlane Token Borsh Schema
+ */
+export class AccountDataWrapper {
+  initialized!: boolean;
+  data!: HyperlaneTokenData;
+  constructor(public readonly fields: any) {
+    Object.assign(this, fields);
+  }
+}
+
+// Should match https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/76467daf3a813952bb84254a09896c0f95046365/rust/sealevel/libraries/hyperlane-sealevel-token/src/accounts.rs#L25C12-L25C26
```

Comment:
Please update the branch name in this reference. It was referring to trevor's branch before it got merged